src/commonTest/kotlin/app/cash/turbine/FlowTest.kt

```diff
@@ -334,8 +334,10 @@ class FlowTest {
   }
 
   @Test fun expectCompleteButWasErrorThrows() = runTest {
-    emptyFlow<Nothing>().test {
-      awaitComplete()
+    assertFailsWith<AssertionError> {
+      flow<Nothing>{ throw RuntimeException() }.test {
+        awaitComplete()
+      }
     }
```

Comment:
Can we copy what `expectItemButWasErrorThrows` is doing above to ensure the message and cause are correct?